README.md

```diff
@@ -16,16 +14,20 @@ To compile the client:
 Now you should have a binary called `push-api-client`.
 
 
-If you want to reinstall the library dependencies, do:
-
-`$ glide install`
+## Running
 
-This creates the `vendor` directory with all the dependencies.
+See [the subscription documentation](https://abiosgaming.com/docs/en/push-api/introduction/overview#message-envelope) for information about how to write subscription specifications.
 
+### With Atlas/v3 API authentication secret 
 
-## Running
 To start the client do:
 
- `$ ./push-api-client --client-id=$CLIENT_ID --client-secret=$CLIENT_SECRET --subscription-file=sample_subscription.json`
+ `$ ./push-api-client --secret=$CLIENT_SECRET --subscription-file=sample_subscription_v3.json`
+
+where `CLIENT_SECRET` is the same that you already use to access v3 of the abios REST API . The `sample_subscription_v3.json` file contains a simple subscription specification that will listen to all events from the `series_updates` channel (for the games your account has access to). 
```

Comment:
`where `CLIENT_SECRET` is the same that you already use to access v3 of the Abios REST API.`

Reply:
Fixed